dist/selleckt-legacy.js

```diff
@@ -25,22 +25,28 @@ module.exports = KEY_CODES;
 */
 'use strict';
 
-var MicroEvent  = function(){};
+var MicroEvent = function(){};
 MicroEvent.prototype = {
-    bind    : function(event, fct){
+    on: function(event, fct){
         this._events = this._events || {};
-        this._events[event] = this._events[event]   || [];
+        this._events[event] = this._events[event] || [];
         this._events[event].push(fct);
     },
-    unbind  : function(event, fct){
+    bind: this.on,
```

Comment:
I'd just go with the

```javascript
function a() {}

var MicroEvent

var MicroEvent.prototype.a = a:
```

pattern so you can just assign the function to two things on the prototype.